src/emulator/host/src/host.cpp

```diff
@@ -82,15 +101,19 @@ void call_import(HostState &host, CPUState &cpu, uint32_t nid, SceUID thread_id)
         // LLE - directly run ARM code imported from some loaded module
 
         if (LOG_IMPORT_CALLS) {
-            const char *const name = import_name(nid);
-            LOG_TRACE("THREAD_ID {} EXPORTED NID {} at {} ({})) called", thread_id, log_hex(nid), log_hex(export_pc), name);
+            const std::unordered_set<uint32_t> lle_nid_blacklist = {};
+
+            log_import_call('L', nid, thread_id, lle_nid_blacklist);
         }
         const ThreadStatePtr thread = lock_and_find(thread_id, host.kernel.threads, host.kernel.mutex);
         const std::lock_guard<std::mutex> lock(thread->mutex);
         write_pc(*thread->cpu, export_pc);
     }
 }
 
-void log_import_calls(bool enabled) {
+#ifndef NDEBUG
+// Import logging is really slow and this allows us to enable/disable it from whenever we please, for easy in debugging
+void set_log_import_calls(bool enabled) {
```

Comment:
Dunno if this should be static or not hehe

Reply:
probably not, it's exported in the header since we want to use it from anywhere (and `static` funcs in header are weird)

Reply:
Okay, I was just not sure